# AutoTest results for test_store (04/02/2013 11:04)
TEST_DATA_CORRECTNESS.test_quotes_in_string: pass
TEST_DATA_CORRECTNESS.test_trailing_blanks: pass
TEST_ERROR_MESSAGE.test_error_message: pass
TEST_ERROR_MESSAGE.test_error_recovery: pass
TEST_EXTENDED_SELECT.test_extended_select: pass
TEST_GUID_TYPE.test_uuid_type: pass
TEST_LENGTH_OF_COLUMN_NAME.test: pass
TEST_MULTIPLE_THREAD.test_multiple_thread: unresolved (Evaluator Died)
	The evaluator process which was executing test TEST_MULTIPLE_THREAD.test_multiple_thread terminated unexpectedly with exit code 1. Any output from the
	evaluator is printed below.
	
	

	test_store: system execution failed.

	Following is the set of recorded exceptions:

	

	******************************** Thread exception *****************************

	In thread           Child thread           0x1AC (thread id)

	*******************************************************************************

	-------------------------------------------------------------------------------

	Class / Object      Routine                Nature of exception           Effect

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute_action @5      result_expected:

	<00000000025B6E58>                         Assertion violated.           Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute_action @5      

	<00000000025B6E58>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DB_SELECTION        load_result @27                                     

	<00000000026DE1E8>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    make_selection @3      

	<00000000025B6E58>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute @16            

	<00000000025B6E58>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    thr_main @3            

	<00000000025B6E58>  (From THREAD)          Routine failure.              Rescue

	-------------------------------------------------------------------------------

	EQA_EVALUATOR       root's creation                                     

	<00000000025505E8>                         Routine failure.              Exit

	-------------------------------------------------------------------------------

TEST_NESTED_TRANSACTION.test: pass
TEST_NUMERIC_TYPES.test_numeric_types: pass
TEST_NUMERIC_TYPES.test_numeric_types_use_decimal_mode: FAIL (Result is not expected)
	on_prepare: ok
	test routine: exceptional (Developer exception in EQA_ASSERTIONS.assert)
		******************************** Thread exception *****************************
		In thread           Root thread            0x0 (thread id)
		*******************************************************************************
		-------------------------------------------------------------------------------
		Class / Object      Routine                Nature of exception           Effect
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        Result is not expected:
		<000000000261F678>                         Developer exception.          Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        
		<000000000261F678>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    raise @11              
		<000000000261F678>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		DEVELOPER_EXCEPTION raise @2                                            
		<000000000261F5C8>  (From EXCEPTION)       Routine failure.              Fail
		-------------------------------------------------------------------------------
		EQA_ASSERTIONS      assert @7                                           
		<00000000025F6888>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  assert @2                                           
		<00000000025F67E8>  (From EQA_TEST_SET)    Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  numeric_types_with_decimal_make_selection @17
		<00000000025F67E8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  test_numeric_types_use_decimal_mode @8
		<00000000025F67E8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
	on_clean: ok
TEST_PROCEDURE.test_procedure: pass
TEST_SELECT.test_basic_select: pass
TEST_SELECT.test_large_string_select: pass
TEST_TRANSACTION.test: pass
